SocialCareCaseViewerApi.Tests/V1/UseCase/CaseStatus/CaseStatusExecutePostUseCaseTests.cs

```diff
@@ -0,0 +1,89 @@
+using FluentAssertions;
+using Moq;
+using NUnit.Framework;
+using SocialCareCaseViewerApi.Tests.V1.Helpers;
+using SocialCareCaseViewerApi.V1.Boundary.Requests;
+using SocialCareCaseViewerApi.V1.Exceptions;
+using SocialCareCaseViewerApi.V1.Gateways;
+using SocialCareCaseViewerApi.V1.Infrastructure;
+using SocialCareCaseViewerApi.V1.UseCase;
+using System.Collections.Generic;
+using System;
+
+namespace SocialCareCaseViewerApi.Tests.V1.UseCase.Relationships
+{
+    [TestFixture]
+    public class CaseStatusExecutePostUseCaseTests
+    {
+        private Mock<IDatabaseGateway> _mockDatabaseGateway;
+        private CaseStatusesUseCase _caseStatusesUseCase;
+        private CreateCaseStatusRequest _request;
+        private readonly CaseStatusType _typeInRequest = TestHelpers.CreateCaseStatusType();
+
+        [SetUp]
+        public void SetUp()
+        {
+            _mockDatabaseGateway = new Mock<IDatabaseGateway>();
+            _caseStatusesUseCase = new CaseStatusesUseCase(_mockDatabaseGateway.Object);
+
+            _mockDatabaseGateway.Setup(x => x.GetCaseStatusTypeWithFields(_typeInRequest.Name))
+                .Returns(_typeInRequest);
+
+            _request = CaseStatusHelper.CreateCaseStatusRequest(type: _typeInRequest.Name);
+
+            _mockDatabaseGateway.Setup(x => x.GetPersonByMosaicId(It.IsAny<int>()))
+                .Returns(TestHelpers.CreatePerson((int) _request.PersonId));
```

Comment:
Why not update TestHelper.CreatePerson to expect a long for the id? The Person class uses a long for the Id field

Reply:
Changed function parameter from int to long